vxfreeswitch/voice.py

```diff
@@ -443,10 +443,12 @@ def send_outbound_message(self, client, message):
         # Wait if call isn't answered
         if self._unanswered_channels.get(client.get_address()):
             try:
-                yield self._unanswered_channels.pop(client.get_address())
+                yield self._unanswered_channels.get(client.get_address())
+                self._unanswered_channels.pop(client.get_address())
             except FreeSwitchClientError:
                 yield self.publish_nack(
                     message['message_id'], 'Unanswered Call')
+                self._unanswered_channels.pop(client.get_address())
```

Comment:
Resolved in https://github.com/praekelt/vumi-freeswitch-esl/pull/32/commits/70e2ecf2c8fc718a11f3109b1757fa85dd737698